### ISSUE TITLE
Tips about your account should be filtered

### ISSUE THREAD

**Issue by kanytu** [Neamar/teamward-client#25]
When showing the game tips, I get tips about my own account ex:
One Trick - Careful about this player because he knows the champion quite well. 

hint: OtpTipBuilder.java:21

**Comment by Neamar** [comment]
Hey,

I figured it might be interesting because it gives you an idea of "what is a one trick pony", since you can compare with yourself. I know why I might be a Katarina OTP, and I can assume that any other OTP will have the same investment on their champion.

Not sure it makes sense, to be fair.

**Comment by kanytu** [comment]
I agree it's interesting to show that. But in my option the string should be changed then. 
For example, my team mates can be OTP but why should I "be careful" about that? You understand?

 It's cool to show your own OTP tip but the string sometimes can be misleading. I keep getting OTP tips for my team mates saying to be careful and I always need to double check if they are on my team or not. Maybe a border for enemy or team mate, along with a different string.

 It's just a suggestion tho :)

**Comment by Neamar** [comment]
You're right, displaying a border with team color would be quite helpful :)

 I'll try to have a look this weekend, unless you wanna try? PR are welcome!

**Comment by kanytu** [comment]
Good to know you're accepting PR :)
I'll see if I have the time to give you a hand.

**Comment by Neamar** [comment]
Hi there! Do you have time for this? Otherwise I'll add it to my backlog,
but I'm working on something big already so it'll take some time!

**Comment by kanytu** [comment]
Hey!

Yes I can do it. Not sure when but I hope soon. When are you planing to tag the next release? Also, I think I might change some strings. How are you dealing with new strings in languages that other contributors translated?

**Comment by Neamar** [comment]
You can change all the strings you want, and the translation system will pick it from there :)

Ideally, if you change the number of placeholders though (`$1%s`), use a new name, otherwise it gets messy.

Looking forward to your changes!
